app/controllers/admin/republishing_controller.rb

```diff
@@ -9,9 +9,26 @@ def republish_past_prime_ministers_index
     redirect_to(admin_republishing_index_path)
   end
 
+  def confirm
+    republishable_document = republishable_documents.find { |document| document[:slug] == params[:document_slug] }
+
+    return render "admin/errors/not_found", status: :not_found unless republishable_document
+
+    @document_title = republishable_document[:title]
+    @republishing_path = republishable_document[:path]
+  end
+
 private
 
   def enforce_permissions!
     enforce_permission!(:republish, Document)
   end
+
+  def republishable_documents
+    [{
+      title: "Past prime ministers",
+      slug: "past-prime-ministers",
+      path: "/government/admin/republishing/republish-past-prime-ministers-index",
```

Comment:
Do we need the path? If we added `-index` to this specific slug, could we reach the path programmatically from the slug (and ideally a path helper, possibly extracted into a helper/util method)? Would that work for other pages?